lib/services/cordova-project-plugins-service.ts

```diff
@@ -707,5 +765,40 @@ export class CordovaProjectPluginsService implements IPluginsService {
 
 		return obj;
 	}
+
+	private _obsoletedIntegratedPlugins: any;
+	private getObsoletedIntegratedPlugins(): IFuture<any> {
+		return (() => {
+			if(!this._obsoletedIntegratedPlugins) {
+				let cordovaJsonContent = this.$fs.readJson(path.join(this.$resources.resolvePath("Cordova"), "cordova.json")).wait();
+				this._obsoletedIntegratedPlugins = cordovaJsonContent && cordovaJsonContent.obsoletedIntegratedPlugins;
+			}
+
+			return this._obsoletedIntegratedPlugins;
+		}).future<any>()();
+	}
+
+	private getObsoletedByPluginIdentifier(pluginIdentifier: string): IFuture<string> {
+		return ((): string => {
+			let obsoletedByInfo = _.find(this.getObsoletedIntegratedPlugins().wait(), (obsoletedPluginInfo: any, key: string) => key.toLowerCase() === pluginIdentifier.toLowerCase()) || Object.create(null);
+			return obsoletedByInfo.obsoletedBy;
```

Comment:
yes, when `undefined` is returned, we consider this as "no, this plugin is not obsoleted by another one".